mass_mailing_sending_queue/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+
+<openerp>
+<data noupdate="1">
+
+<record id="sending_batch_size" model="ir.config_parameter" forcecreate="True">
+    <field name="key">mail.mass_mailing.sending.batch_size</field>
+    <field name="value">500</field>
```

Comment:
This is the number of emails created in each cronjob iteration. Those emails are add to the standard emails queue (mail.mail.process_email_queue), processed by other cronjob. These two queues are independent.

odoo/odoo#6386 PR fix ```process_email_queue```, so I think that this addon doesn't close it.